mnemosyne/core/profiles.py

```
@@ -0,0 +1,707 @@
+        scope = str(settings.get("default_scope", "session"))
+        if scope != "global":
+            errors.append(
+                f"cross_session=1 but default_scope='{scope}' — "
+                f"cross-session visibility requires global scope"
+            )
+
+    # Rule 4: smart_compress implies llm_enabled
+    if _to_bool(settings.get("smart_compress", "1")):
+        if not _to_bool(settings.get("llm_enabled", "true")):
+            errors.append(
+                f"smart_compress=1 but llm_enabled=false — "
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)